webapp/package.json

```diff
@@ -24,7 +24,6 @@
         "react": "16.13.1",
         "react-bootstrap": "1.3.0",
         "react-custom-scrollbars": "4.2.1",
-        "react-dom": "18.2.0",
```

Comment:
Wondering why this was removed. Do you think it's being used by anything? I suppose the bundler would complain if so. I see it's still present in `package-lock.json`, but no package there depends on it

Reply:
@ micmister I think it was removed because of duplicity. It's still present on line number `17`.